Dockerfile

```diff
@@ -770,8 +772,8 @@ RUN \
   --disable-ffplay \
   --enable-static \
   --enable-gpl \
-  --enable-version3 \
-  --enable-nonfree \
+    --enable-version3 \
```

Comment:
weird indent? i guess we use auto formatters with different opinions? :)

Reply:
Ah missed that, will fix.